suite/src/__tests__/fast/get-events.test.ts

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, test } from '@jest/globals'
+import fetch from 'cross-fetch'
+import { generateRandomEventId, generateRandomEvent } from '../../utils/rustCeramicHelpers'
+
+const CeramicUrls = String(process.env.CERAMIC_URLS).split(',')
+
+async function postEvent(url: string, event: any) {
+  let response = await fetch(url + '/ceramic/events', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(event),
+  })
+  if (!response.ok) {
+    throw new Error(`HTTP error! status: ${response.status}`)
+  }
+}
+
+async function getEventData(url: string, eventId: string) {
+  let response = await fetch(url + `/ceramic/events/${eventId}`)
+  console.log(response)
+  return response
+}
+
+describe('rust-ceramic e2e test', () => {
+  const ceramicUrl = CeramicUrls[0]
+  test('post and get event data, success', async () => {
+    const eventId = generateRandomEventId()
+    const event = generateRandomEvent(eventId)
+    // publishing the event to rust-ceramic
+    await postEvent(ceramicUrl, event)
+    // fetching the event from its event-id from rust-ceramic
+    const getResponse = await getEventData(ceramicUrl, eventId)
+    expect(getResponse.status).toEqual(200)
+    expect(await getResponse.json()).toEqual({
+      eventId: eventId,
```

Comment:
Assuming my [changes land](https://github.com/ceramicnetwork/rust-ceramic/pull/255), I modified the event response to be `{id, data}` to save some bytes based on review. This only applies to responses (so the `GET event/$id` and `GET feed/events` endpoints). The input is still `eventId/eventData` to avoid breaking existing functionality.